backend/src/dao/ShoutoutsDao.ts

```diff
@@ -0,0 +1,33 @@
+import { memberCollection, shoutoutCollection } from '../firebase';
+import { Shoutout, DBShoutout } from '../DataTypes';
+
+export default class ShoutoutsDao {
+  static async getShoutouts(email: string, type: string): Promise<Shoutout[]> {
```

Comment:
Use literal type for `type`: `type: 'given' | 'received'` Same with the frontend one.